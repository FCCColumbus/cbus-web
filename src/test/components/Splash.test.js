import renderer from "react-test-renderer";

import Splash from "../../Splash";

describe("Splash", () => {
    it("renders", () => {
        const view = renderer.create(<Splash />);
        expect(view).toMatchSnapshot();
    });
});
