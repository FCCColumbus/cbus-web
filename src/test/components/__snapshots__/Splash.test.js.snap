// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash renders 1`] = `
<div
  className="splash"
>
  <div
    className="byline"
  >
    <h2>
      Free Code Camp is the local chapter of FCC, the best way to learn coding for free!
    </h2>
  </div>
  <img
    alt="people having fun coding"
    src="/cottonbro-studio.jpg"
  />
</div>
`;
